UI/cypress/e2e/contactus.feature

```diff
@@ -0,0 +1,21 @@
+﻿ Feature: Contactus Page Tests
+   Scenario: contact us page url is '/Contactus'
+     Given a user has arrived on the contact us page
+    Then the page URL ends with '/Contactus'
+
+   Scenario: page as link to back button
```

Comment:
Please add the following two scenarios:

- Scenario: contact us page title is 'Contact us'
- Scenario: user clicks service name on contact us page